### ISSUE TITLE
do this work batch input>?

### ISSUE THREAD

**Issue by vahidalvandi** [GrahamCampbell/Security-Core#9]
hi
i want to clear all array() input with Security::create()->clean

**Comment by GrahamCampbell** [comment]
Yes, you can pass an array of strings. No nesting is supported. Purely a list of string or a string is allowed.
